src/main.rs

```diff
@@ -67,7 +67,7 @@ fn main() {
                 err = arc.inner();
             }
             if let Some(e) = err.downcast_ref::<edgedb_errors::Error>() {
-                print::edgedb_error(e, true);
+                print::edgedb_error(e, false);
```

Comment:
This is the "verbose" argument.

Also, I consider boolean arguments to functions to be an anti-pattern in languages without named arguments. We should probably make an enum for opting in to Verbose.